### ISSUE TITLE
Platform bindings for min/max pitch setting

### ISSUE THREAD

**Issue by chloekraw** [mapbox/mapbox-gl-native-ios#176]
https://github.com/mapbox/mapbox-gl-native/pull/16219

Would be great to get this in before the next branch cut.

cc @julianrex @jmalanen

**Comment by 1ec5** [comment]
Unlike `MGLMapView.pitchEnabled`, the proposed options appear to affect programmatic camera changes as well as user gestures. As such, they would imitate (and be redundant to) the current workflow of implementing `-[MGLMapViewDelegate mapView:shouldChangeFromCamera:toCamera:]` like this:

```objc
- (BOOL)mapView:(MGLMapView *)mapView shouldChangeFromCamera:(MGLMapCamera *)oldCamera toCamera:(MGLMapCamera *)newCamera {
    return newCamera.pitch <= 45;
}
```

**Comment by 1ec5** [comment]
mapbox/mapbox-gl-native#16219 was released in gl-native v1.3.0, so this feature depends on #181.

**Comment by chloekraw** [comment]
Amending the OP: this is now a beta release blocker.
